spec/features/admin/site_customization/pages_spec.rb

```diff
@@ -11,7 +11,7 @@
     custom_page = create(:site_customization_page)
     visit admin_site_customization_pages_path
 
-    expect(page).to have_content(custom_page.title)
```

Comment:
As we decided to keep the title and the slug on `/admin/site_customization/pages/index.html.erb` could you keep both also in the spec? 🙏 😌 

Thanks!

Reply:
Of course @decabeza! I already added in all scenarios in spec. I forgot to update the spec. 🐰 Thanks!!